src/bindings/python/flux/job/executor.py

```diff
@@ -40,6 +40,7 @@ class _FluxExecutorThread(threading.Thread):
     # pylint: disable=too-many-arguments
     def __init__(
         self,
+        broken_event,
```

Comment:
Thoughts on making this an optional argument, and if it is not provided, just automatically instantiating a new `threading.Event()`?  Seems like that might clean up some of the new test code.

Reply:
It definitely would. If you ignore the tests, though, I think the intention could be unclear, since ``FluxExecutor`` always passes all of the arguments to its threads---although it could always be explained with a comment. So on balance I'm ambivalent.

Reply:
Sounds good. Happy to leave as is.